asm/code_8098BDC.s

```diff
@@ -5573,7 +5573,7 @@ _0809BC26:
 	lsrs r4, 31
 	bl sub_809B294
 	adds r0, r4, 0
-	bl CreateKangaskhanStorage
```

Comment:
Per Discord conversation, let's keep the existing naming scheme here.

Reply:
Addressed